doc/code_snippets/snippets/connectors/go/hello.go

```diff
@@ -0,0 +1,132 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"github.com/tarantool/go-tarantool/v2"
+	_ "github.com/tarantool/go-tarantool/v2/datetime"
+	_ "github.com/tarantool/go-tarantool/v2/decimal"
+	_ "github.com/tarantool/go-tarantool/v2/uuid"
+	"time"
+)
+
+func main() {
+	ctx, cancel := context.WithTimeout(context.Background(), time.Second)
```

Comment:
Nit: you can add a comment `// Connect to the database` to make this source file more readable even without the doc page.